apps/src/code-studio/pd/application_dashboard/principal_approval_buttons.jsx

```diff
@@ -52,6 +54,10 @@ export default class PrincipalApprovalButtons extends React.Component {
         this.props.applicationId,
         data.principal_approval
       );
+      this.setState({
+        sendEmailRequest: null,
+        showSendEmailButton: false
```

Comment:
would we still want to show this button so they can resend if desired?

Reply:
I think we shouldn't but lets see how people like it in the bug bash